docs/parameters.md

```diff
@@ -60,6 +60,23 @@ There are multiple use cases from the users perspective that dictate what parame
 | foreman-installer Parameter | Description | Reason |
 | --------------------------- | ----------- | ------ |
 
+#### Logging
+
+##### Mapped
+
+| Parameter | Description | foreman-installer Parameter |
+| ----------| ----------- | --------------------------- |
+| `--foreman-logging-level` | Allows setting the level of the Foreman logs | `--foreman-logging-level` |
+| `--foreman-logging-layout` | Allow defining a different style of structured output for Foreman logs | `--foreman-logging-layout` |
```

Comment:
In our current module we derive the default layout on the logging type (https://github.com/theforeman/puppet-foreman/blob/86e73d267bb6e05eb607ae8fedc00a8ffa132334/manifests/config.pp#L38-L44). My feeling is that this should be good enough for most users. Not sure we need to expose this.

Reply:
You are thinking we should just hard-code this to `multiline_request_pattern` and not give users the opportunity to change it? I was figuring `json` might be useful for users that ship logs to a centralized logging system.

Reply:
Then we may also need to makevthe type configureable. Something to investigate

Reply:
For now, I am going to put this into the unmapped category and leave it hard-coded. We can address stdout vs journald in the future and do so via https://github.com/theforeman/foreman-quadlet/issues/21